controllers/storagecluster/cephtoolbox.go

```diff
@@ -72,6 +74,29 @@ func (r *StorageClusterReconciler) ensureToolsDeployment(sc *ocsv1.StorageCluste
 			return err
 		}
 
+		// Add multus related annotations, if multus provider is set
+		if isMultus(sc.Spec.Network) {
+			err := validateMultusSelectors(sc.Spec.Network.Selectors)
+			if err != nil {
+				return err
+			}
+
+			multusNetNs, err := util.GetOperatorNamespace()
+			if err != nil {
+				return err
+			}
+			multusNetName := ""
+			if sc.Spec.Network.Selectors["public"] != "" {
+				multusNetName = sc.Spec.Network.Selectors["public"]
+			} else if sc.Spec.Network.Selectors["cluster"] != "" {
+				multusNetName = sc.Spec.Network.Selectors["cluster"]
+			}
```

Comment:
Can't we have both? Is it always either "public" or "cluster"?

Reply:
TBH, I am not sure. I was trying to figure out the annotation required and stumbled across this [piece](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/ocs/utils.py#L839) of code in ocs-ci, and just replicated that here.
Let me ask around someone from rook to confirm this.

Reply:
Multus can be public, cluster, or both.

Reply:
I would probably suggest a new helper method like this:

```go
func getMultusPublicNetwork(/* ... */) (string, error) {
   err := validateMultusSelectors(sc.Spec.Network.Selectors)
			if err != nil {
				return "", err
			}

			multusNetNs, err := util.GetOperatorNamespace()
			if err != nil {
				return "", err
			}
			multusNetName := ""
			if sc.Spec.Network.Selectors["public"] == "" {
				return "", nil
			}
			if sc.Spec.Network.Selectors["public"] != "" {
				multusNetName = sc.Spec.Network.Selectors["public"]
			}
			return fmt.Sprintf("%s/%s", multusNetNs, multusNetName), nil

}


# ...


net, err := getMultusPublicNetwork(/* ... */)
if err != nil {
	return err
}
if net != "" {
	# TODO: apply annotation to pod
}
```

Reply:
Thanks for the review, and the helper method. Updated the code accordingly.